deps/src/polymake_integers.cpp

```diff
@@ -0,0 +1,36 @@
+#include "polymake_includes.h"
+
+#include "polymake_tools.h"
+
+#include "polymake_functions.h"
+
+#include "polymake_integers.h"
+
+pm::Integer cast_bigint_to_integer(jl_value_t* integer){
+    pm::Integer* p;
+    p = reinterpret_cast<pm::Integer*>(integer);
+    return *p;
+}
+
+void polymake_module_add_integer(jlcxx::Module& polymake){
+  polymake.add_type<pm::Integer>("pm_Integer", jlcxx::julia_type("Integer", "Base"))
+    .constructor<int32_t>()
+    .constructor<int64_t>()
+    // the symmetric definitions are on the julia side
+    .method("+",[](pm::Integer &a, pm::Integer &b){ return a + b; })
+    .method("+",[](pm::Integer &a, int64_t b){ return a + static_cast<long>(b); })
+    .method("+",[](pm::Integer &a, int32_t b){ return a + b; })
+    // the symmetric definitions are on the julia side
```

Comment:
While I am technically fine with this, as they seem to be magically there, there are two potential problems here:
1. This adds another indirection in function calls. While Julia should technically be able to resolve this, we should look at the assembly if it really gets rid of the function call. Otherwise simply add the symmetric devs here, since this should cause stupid performance decreases otherwise.
2. Should we have a special addition for BigInt? Currently, Adding a `pm_Integer` and a `BigInt` creates a new `pm_Integer` from the `BigInt`. While `BigInt`s are not that popular anyway, this could also potentially cause strange performance penalties.